### ISSUE TITLE
abbreviation expansion is not always useful or helpful

### ISSUE THREAD

**Issue by stripathy** [renaud/neuroNER#9]
GABA is fine as is, but it's not helpful to expand it to whatever the author claims it stands for.

**Comment by renaud** [comment]
Could you come up with an exaustive list of abrevs that do not need to be expanded? If so, I could "whitelist" them.
Note to self: the relavant code is [here](https://github.com/BlueBrain/bluima/blob/fcb661f959698c5d5f07e0bd1e3b8448ccef0093/modules/bluima_abbreviations/src/main/java/ch/epfl/bbp/uima/AbbreviationExpander.java).

**Comment by stripathy** [comment]
Yes - I can do this, but it'll take some time. We should prioritize this in our next meeting.
